src/commands/admin/shutdown.ts

```diff
@@ -1,15 +1,21 @@
 import { BOT } from '@root/config';
-import { botMasterPerms } from '@lib/permissions';
-import { Message } from 'discord.js';
+import { BOTMASTER_PERMS } from '@lib/permissions';
+import { ApplicationCommandPermissionData, CommandInteraction, Message } from 'discord.js';
 import { Command } from '@lib/types/Command';
 
 export default class extends Command {
```

Comment:
This command kind of blows me up because `systemd` controls sage and auto-restarts it whenever it stops... We might just rename this command to `restart` and call it a day because that's essentially what it would do